packages/create-sitecore-jss/src/templates/nextjs/scripts/config/plugins/computed.ts

```diff
@@ -7,11 +7,12 @@ import { ConfigPlugin, JssConfig } from '..';
  */
 class ComputedPlugin implements ConfigPlugin {
   // should come after other plugins (but before fallback)
-  order = 10;
+  order = 3;
```

Comment:
Let's leave it as _10_ how it was because in case we will add more plugins, it will be more complex to set a new plugin before _computed_. So, just set _multisite_ plugin order as _11_